massdriver.yaml

```diff
@@ -230,71 +229,15 @@ params:
           maxItems: 1
           items:
             type: string
-    observability:
-      type: object
-      title: Observability
-      description: Configure logging and metrics collection and delivery for your entire cluster
-      required:
-        -  logging
-      properties:
-        logging:
-          type: object
-          title: Logging
-          description: Configure logging for your cluster
-          required:
-            - destination
-          properties:
-            destination:
-              type: string
-              title: Destination
-              description: Where to send logs
-              default: disabled
-              oneOf:
-              - title: OpenSearch (in cluster)
-                const: opensearch
-              - title: Disabled
-                const: disabled
-          dependencies:
-            destination:
-              oneOf:
-                - properties:
-                    destination:
-                      const: opensearch
-                    opensearch:
-                      type: object
-                      title: OpenSearch
-                      required:
-                        - persistence_size
-                        - retention_days
-                      properties:
-                        persistence_size:
-                          type: integer
-                          title: Persistence Size GiB
-                          description: Size of the persistent volume to use for OpenSearch
-                          minimum: 1
-                          maximum: 10000
-                          default: 10
-                        retention_days:
-                          type: integer
-                          title: Retention Days
-                          description: Number of days to retain logs and metrics in an OpenSearch Index
-                          minimum: 1
-                          maximum: 365
-                          default: 7
-                  required:
-                    - opensearch
-                - properties:
-                    destination:
-                      const: disabled
 
 connections:
   required:
     - azure_service_principal
-    - vnet
```

Comment:
Renaming this connection is a breaking change. Theres no way for the system to know that is a rename as opposed to a delete and recreate. This means that any existing bundles with a link between a vnet and an AKS cluster will have that link deleted.

Reply:
Are there currently any deployed AKS instances? If so, I'll remove this change from the PR.